catalog/cataloger_create_entries.go

```diff
@@ -33,9 +36,30 @@ func (c *cataloger) CreateEntries(ctx context.Context, repository, branch string
 		if err != nil {
 			return nil, err
 		}
-		for i := range entries {
-			if _, err := insertEntry(tx, branchID, &entries[i]); err != nil {
-				return nil, fmt.Errorf("entry at %d: %w", i, err)
+		// single insert per batch
+		for i := 0; i < len(entries); i += c.CreateEntriesInsertSize {
+			sqInsert := psql.Insert("catalog_entries").
+				Columns("branch_id", "path", "physical_address", "checksum", "size", "metadata", "creation_date", "is_expired")
+			j := i + c.CreateEntriesInsertSize
+			if j > len(entries) {
+				j = len(entries)
+			}
+			for _, entry := range entries[i:j] {
+				creationDate := entry.CreationDate
+				if entry.CreationDate.IsZero() {
+					creationDate = time.Now()
```

Comment:
This is a different clock than on the database.  I think we should use "now" on the database, like we now [do on CreateEntry](https://github.com/treeverse/lakeFS/blob/8b28f3affda313c96afd385a8c58e54e813e6ea3/catalog/cataloger_create_entry.go#L56).